config/webpack.common.js

```diff
@@ -5,11 +5,12 @@ const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 const Dotenv = require('dotenv-webpack');
 require('dotenv').config(); // For runtime-env-vars.js
```

Comment:
I think we can totally remove the node `dotenv` which we're using for `_env_encoded`, since the new virtMeta is replacing its purpose. We should `yarn remove dotenv` and get rid of this import (`dotenv-webpack` will still use it internally but we don't need to directly depend on it anymore)